### ISSUE TITLE
Use redis hash to store ragots

### ISSUE THREAD

**Issue by barodeur** [ensiie/ragots#29]

**Comment by Soulou** [comment]
I didn't know redis manages hashes, so why not, it will be cleaner than simple ragot:ID:attribute entry

**Comment by barodeur** [comment]
Yep, I thought it was a good idea tu use `ragot:ID:attribute` for educational purpose during tIIch, but at the end of the day, it's more complicated to maintain and less performant.
